### PR TITLE
Desktop: Resolves #9998: Fixed text wrapping in Spellcheck button

### LINKED ISSUE #9998: Text in the spellcheck button is wrapping
### Operating system

macOS

### Joplin version

2.13.15

### Desktop version info

Joplin 2.13.15 (prod, darwin)

Client ID: c08e549cb29947a58cf80d908c0a74a3
Sync Version: 3
Profile Version: 44
Keychain Supported: No

Revision: 7d2c1c0

### Current behaviour

The current behavior is that when I add multiple spellcheck languages, the button's text wraps as shown in two pictures below.

Screenshot 2024-02-25 at 11 08 39

Screenshot 2024-02-25 at 11 09 42

### Expected behaviour

The text should not wrap. Here are suggested solutions:

#### Solution 1:
```css
someElement {
  .
  .
  .
  white-space: nowrap; 
  overflow: hidden;
  text-overflow: ellipsis;
  
}
```

#### Solution 2:
```css
someElement {
  .
  .
  .
  white-space: nowrap; 
  overflow-x: scroll;
  &::-webkit-scrollbar {
       display: none;
  }
  
}
```

### Logs

_No response_

### PR DESCRIPTION
**Pull request by RadCod3**
### Description
Fixed https://github.com/laurent22/joplin/issues/9998 using one liner.
### Screenshots
Before:
image
After:
image

### CODE REVIEW

**Comment by RadCod3** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by laurent22** [comment]
Now that I see the screenshot... Is it possible to instead cut the text with an ellipsis?

**Comment by RadCod3** [comment]
Will try to do so and  get back to you.

**Comment by RadCod3** [comment]
I tried the following css but the text won't truncate without setting a specific value to the width, which then wont show the full text even when you expand it. Will try it out some more, any ideas?
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
display: block;
```
Edit:
I've found some references: https://css-tricks.com/flexbox-truncated-text/

Edit 2: Got a working solution now! Is this acceptable?

**Comment by RadCod3** [comment]
I had to wrap text inside the ToolbarButtons with a span, otherwise it wouldn’t truncate with ellipsis, it just truncated from both sides.

Also apologies if I do this wrong but @PackElend label me please (Do I ask for this label after creating future PRs as well?).

**Comment by PackElend** [comment]
> (Do I ask for this label after creating future PRs as well?)

yes, otherwise we cannot track your work

File: packages/app-desktop/gui/ToolbarButton/ToolbarButton.tsx [hunk 1]
```diff
@@ -43,7 +43,10 @@ export default function ToolbarButton(props: Props) {
 	if (!isEnabled) classes.push('disabled');
 
 	const onClick = getProp(props, 'onClick');
-
+	const style: any = {
```
Comment (laurent22) [inline packages/app-desktop/gui/ToolbarButton/ToolbarButton.tsx]: Please don't use "any". `React.CSSProperties` is the correct type here I believe

**Pull request merged**